### DISCUSSION TITLE
Transcode AVC - resolution

### DISCUSSION THREAD

**Discussion by Timmmy** [photoprism/photoprism#4330] [General]
Hi all,

Just doublechecking what is mentioned in the docs: [Docs](https://docs.photoprism.app/getting-started/advanced/transcoding/)
For video's that are already AVC encoded, the docs mention that they will not be re-encoded if the bitrate exceeds the one that is configured. However there is no similar statement for resolution.

Does this imply that AVC encoded videos will be re-encoded if the resolution is higher then the set resolution?

Thanks for clarifying.

Timmmy

**Comment by lastzero** [comment]
Existing AVC videos are generally not reencoded at the moment. There is only one avc file and not multiple files for different resolutions, bitrates, color profiles etc.

**Reply by Timmmy** [reply]
Thanks for the quick reply @lastzero !

I did not mean multiple files or so.
Maybe it's clearer with an example. If limit is set to 720, but a 1080 source file is uploaded. Will this be re-encoded to 720 or will the original file be copied to the sidecar folder. I assume the latter?

I submitted a pull request to clarify this in the docs.
